### DISCUSSION TITLE
How to change the line break of generated html file?

### DISCUSSION THREAD

**Discussion by Saigut** [getpelican/pelican#3143] [Q&A]
- [ Y] I have read the [Filing Issues](https://docs.getpelican.com/en/latest/contribute.html#filing-issues) and subsequent “How to Get Help” sections of the documentation.
- [ Y] I have searched the [issues](https://github.com/getpelican/pelican/issues?q=is%3Aissue) (including closed ones) and believe that this is not a duplicate.

- **OS version and name**: Macos monterey
- **Python version**: 3.9.6
- **Pelican version**: 4.8.0
- **Link to theme**: pelican-cait(customed by myself)
- **Links to plugins**: None
- **Link to your site**: None
- **Link to your source**: None
- **Link to a [Gist](https://gist.github.com/) with the contents of your settings file**: None

## Issue

How to change the line break of generated html file? The line break of generated html files is LF, but I want CRLF, how to change to CRLF? Thanks.

**Comment by justinmayer** [comment]
@Saigut: HTML is generated by either Python-Markdown (for Markdown content) or Docutils (for reStructuredText content), and thus this topic is likely outside the scope of Pelican itself. I suggest you reach out to either of those communities, depending on which source content format(s) you use.
